app/components/primer/beta/auto_complete/item.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <span class="ActionList-content">
+    <% if leading_visual %>
+        <span class="ActionList-item-visual ActionList-item-visual--leading">
+            <%= leading_visual %>
+        </span>
+    <% end %>
+    <% if @description %>
+        <span class="ActionList-item-descriptionWrap ActionList-item-blockDescription">
+            <span class="ActionList-item-label"><%= content %></span>
+            <span class="ActionList-item-description"><%= @description %></span>
```

Comment:
Can someone help me make `@description` a "slot"? I need to be able to apply conditional logic to the contents. @jonrohan @camertron